tests/conftest.py

```diff
@@ -212,3 +212,54 @@ def cfg_physenet(cfg_physenet_global, tmp_path) -> DictConfig:
     yield cfg
 
     GlobalHydra.instance().clear()
+
+
+@pytest.fixture(scope='package')
+def cfg_bcresnet_global() -> DictConfig:
+    """Fixture for providing a global configuration for Breast Cancer Classification with ResNet.
+
+    Returns:
+        DictConfig: The global configuration for Breast Cancer Classification with ResNet.
+    """
+    with initialize(version_base='1.3', config_path='../configs'):
+        cfg = compose(config_name='train.yaml',
+                      return_hydra_config=True, overrides=['model=bcc', 'data=bcc'])
+
+        # set defaults for all tests
+        with open_dict(cfg):
+            cfg.paths.root_dir = str(
+                pyrootutils.find_root(indicator='.project-root'))
+            cfg.trainer.max_epochs = 1
+            cfg.trainer.accelerator = 'cpu'
+            cfg.trainer.devices = 1
+            cfg.data.input_size = [256, 256]
+            cfg.data.num_workers = 0
+            cfg.data.batch_size = 4
+            cfg.data.pin_memory = False
+            cfg.extras.print_config = False
+            cfg.extras.enforce_tags = False
+            cfg.logger = None
+
+    return cfg
+
+
+@pytest.fixture(scope='function')
+def cfg_bcresnet18(cfg_bcresnet_global, tmp_path) -> DictConfig:
+    """Fixture for providing a specific configuration for Breast Cancer Classification with ResNet.
+
+    Args:
```

Comment:
remove the args and returns. The function is quite clear by name and functionality